src/guacscanner/guacscanner.py

```diff
@@ -0,0 +1,548 @@
+"""Query AWS for new (destroyed) instances and add (remove) Guacamole connections for them.
+
+Also check for instances that have been destroyed and remove their
+corresponding connections.
+
+EXIT STATUS
+  0   Update was successful.
+  >0  An error occurred.
+
+Usage:
+  guacscanner [--log-level=LEVEL] [--oneshot] [--sleep=SECONDS] [--postgres-password=PASSWORD|--postgres-password-file=FILENAME] [--private-ssh-key=KEY|--private-ssh-key-file=FILENAME] [--postgres-username=USERNAME|--postgres-username-file=FILENAME] [--rdp-password=PASSWORD|--rdp-password-file=FILENAME] [--rdp-username=USERNAME|--rdp-username-file=FILENAME] [--vnc-password=PASSWORD|--vnc-password-file=FILENAME] [--vnc-username=USERNAME|--vnc-username-file=FILENAME] [--vpc-id=VPC_ID]
+  guacscanner (-h | --help)
+
+Options:
+  -h --help              Show this message.
+  --log-level=LEVEL      If specified, then the log level will be set to
+                         the specified value.  Valid values are "debug", "info",
+                         "warning", "error", and "critical". [default: info]
+  --oneshot              If present then the loop that adds (removes) connections for new (terminated) instances will only be run once.
+  --postgres-password=PASSWORD    If specified then the specified value will be used as the password when connecting to the PostgreSQL database.  Otherwise, the password will be read from a local file.
+  --postgres-password-file=FILENAME    The file from which the PostgreSQL password will be read. [default: /run/secrets/postgres-password]
+  --postgres-username=USERNAME    If specified then the specified value will be used when connecting to the PostgreSQL database.  Otherwise, the username will be read from a local file.
+  --postgres-username-file=FILENAME    The file from which the PostgreSQL username will be read. [default: /run/secrets/postgres-username]
+  --private-ssh-key=KEY  If specified then the specified value will be used for the private ssh key.  Otherwise, the ssh key will be read from a local file.
+  --private-ssh-key-file=FILENAME  The file from which the private ssh key will be read. [default: /run/secrets/private-ssh-key]
```

Comment:
Use preferred capitalization:
```suggestion
  --private-ssh-key-file=FILENAME  The file from which the private SSH key will be read. [default: /run/secrets/private-ssh-key]
```